macs2.wdl

```diff
@@ -29,7 +29,14 @@ task PeakCalling {
         String outDir = "macs2"
         String sampleName
         String format = "AUTO"
-        Boolean nomodel = false
+        Boolean? nomodel
```

Comment:
This boolean just represents whether a flag should be used in the command, so giving this a default of false makes sense. I'm not entirely sure what will happen in line 57 if this is null.

```suggestion
        Boolean nomodel = false
```